bee-node/src/config.rs

```diff
@@ -143,14 +143,7 @@ impl<S: NodeStorageBackend> NodeConfigBuilder<S> {
     }
 
     /// Returns the built node config.
-    pub fn finish(self) -> NodeConfig<S> {
-        // Create the local node entity.
-        let local = if let Some(keypair) = self.identity {
-            Local::from_keypair(keypair, self.alias).expect("error creating local from hex encoded keypair")
-        } else {
-            Local::new(self.alias)
-        };
-
+    pub fn finish(self) -> (Option<String>, bool, NodeConfig<S>) {
```

Comment:
Actually the alias needs to be in the `NodeConfig` it was there through `local` but then it's not anymore. Just add `alias` to the config and you don't have to return it anymore.

Reply:
As for the bool, I don't really like it but I guess it's temporary, we leave it for one release for people to migrate and then we can remove it at some point.